src/shared/rpc/methods/sign-stacks-transaction.ts

```diff
@@ -0,0 +1,40 @@
+import { DefineRpcMethod, RpcRequest, RpcResponse } from '@btckit/types';
+import { StacksNetworks } from '@stacks/network';
+import * as yup from 'yup';
+
+import { formatValidationErrors, getRpcParamErrors, validateRpcParams } from './validation.utils';
+
+const rpcSignStacksTransactionParamsSchema = yup.object().shape({
+  stxAddress: yup.string().required(),
+  serializedTransactionHex: yup.string().required(),
+  attachment: yup.string(),
+  appDetails: yup.object({
+    name: yup.string(),
+    icon: yup.string(),
+  }),
```

Comment:
`appDetails` is an old Stacks request structure. Maybe we can just pass an `icon`?

Imo we should drop `name`. It's dangerous as it can be used to trick users. Honestly, even can the icon. Right now we use the favicon of the `origin`, so maybe we can remove entirely?